src/content/doc-surrealdb/installation/upgrading/2.0.mdx

```diff
@@ -1,37 +1,46 @@
 ---
 sidebar_position: 1
-sidebar_label: Upgrading to 2.0.0
-title: Upgrading to 2.0.0 | Upgrading | Installation
-description: This guide will help you upgrade your current SurrealDB installation to the latest `2.0.0` release.
+sidebar_label: Upgrading from 1.x to 2.x
+title: Upgrading from 1.x to 2.x | Upgrading | Installation
+description: This guide will help you upgrade your current SurrealDB installation to the latest `2.x` release.
 ---
 
-# Upgrading to `2.0.0` 
+# Upgrading from `1.x` to `2.x` 
 
 The `2.0.0` release of SurrealDB includes [many new features, improvements, and bug fixes](https://surrealdb.com/releases#v2-0-0). However, because of changes in the underlying way that SurrealDB stores data, three steps are required to migrate to `2.0.0`. 
 
 We've listed them below, and we advise following this migration guide whether or not you encounter this issue!
 
-## Issues
+## Breaking changes
 
+### Datastore 
 - A change in the underlying way data is stored in SurrealDB was caused by how ranges and record IDs are stored in the database. 
     -  [Record IDs now support storing UUIDs instead of strings](https://github.com/surrealdb/surrealdb/pull/4491). 
     -  [`Value::Range` stored as just a range instead of a string](https://github.com/surrealdb/surrealdb/pull/4506).
 
+### SurrealQL
 - Some breaking changes were made to the SurrealQL language.
     - Behaviour of the `UPDATE` statement which no longer allows to update a record that does not exist. In favor of the [`UPSERT`](/docs/surrealql/statements/upsert) statement.
     - Deprecated `file://` in favour of `rocksdb://` or the new `surrealkv://`. 
     - Renamed functions [(meta::*)](/docs/surrealql/functions/database/meta) to (record::*)
     - Functions `string::endsWith()` and `string::startsWith()` renamed to `string::ends_with()` and `string::starts_with()`
+### Authentication & Headers
+    - Authentication is now enabled by default as you previously would with `--auth`. The [`--unauthenticated`]() flag is now required in order to provide the previous default behavior.
+    - Specifying the level on which credentials will be authenticated is now required when connecting to SurrealDB. By default, this level will be root. This can be provided with the `--auth-level` flag in the CLI or the surreal-auth-ns and surreal-auth-db headers in the HTTP REST API.
```

Comment:
```suggestion
    - Specifying the level on which credentials will be authenticated is now required when connecting to SurrealDB. By default, this level will be root. This can be provided with the `--auth-level` flag in the CLI or the `surreal-auth-ns` and `surreal-auth-db` headers in the HTTP REST API.
```